mlflow/pyfunc/__init__.py

```diff
@@ -372,6 +372,19 @@ def predict(self, data: PyFuncInput) -> PyFuncOutput:
             data = _enforce_schema(data, input_schema)
         return self._predict_fn(data)
 
+    def unwrap_python_model(self, unwrap_metadata: bool = False):
+        """
+        Unwrap the underlying Python model object, and it's metadata (optionally).
+
+        :param unwrap_metadata: (optional) If True, return a tuple of the
+        underlying Python model object and metadata loaded from the MLmodel file.
+        """
+        return (
```

Comment:
Instead of a mixed return type (object vs. tuple) can you convert this return to a named tuple with a default of unwrap_metadata=False having a metadata value = None. That way there is consistency in the returned values of the model object. 
```python
my_random_forest_model = pyfunc_model.model
```

Reply:
On second thought, let's leave this out completely. Let's restrict this functionality to strictly just returning the model object and leave the metadata to further PR work if needed.